packages/odyssey-react/src/components/Banner/Banner.theme.ts

```diff
@@ -24,7 +24,7 @@ export const theme: ThemeReducer = (theme) => ({
   IconMarginInlineEnd: theme.SpaceScale3,
   IconSize: theme.FontSizeHeading5,
 
-  HeadingMarginInlineEnd: theme.SpaceScale0,
+  HeadingMarginInlineEnd: theme.SpaceScale1,
```

Comment:
This is a non-breaking design change to give the heading some more breathing room.